packages/cardano-services/src/ChainHistory/DbSyncChainHistory/mappers.ts

```diff
@@ -209,7 +221,7 @@ export const mapCertificate = (
       __typename: Cardano.CertificateType.PoolRegistration,
       cert_index: certModel.cert_index,
       deposit: BigInt(certModel.deposit),
-      poolParameters: null as unknown as Cardano.PoolParameters
+      poolParameters: mapPoolParameters(certModel)
```

Comment:
This looks like a fairly cheap change, so just wondering: are we using/planning to use it for something in Lace?

Reply:
IDK, I just saw `getCertificatesByType` is not able to pick stake pool registration certificates and I fixed it.